### PR TITLE
Final Refactor

### LINKED ISSUE #35: movie-app final refactor
**Is your feature request related to a problem? Please describe.**
Take a final look through all of movie app code, and refactor/remove non necessary code for better future maintainability.

### PR DESCRIPTION
**Pull request by BlackCubes**
## Changes
1. Refactored the entire application using prettier.
2. Added a prettier config file.
3. Created custom logos.
4. Refactored the manifest.json file.
5. Refactored the index.html file

## Purpose
The purpose of this feature is to a final refactorization along with making custom logos.

## Approach
The approach is to use prettier with a config file to format the entire code. Another approach is to use Figma to create the custom logos.

Closes #35

### CODE REVIEW

File: movie-app/public/index.html [hunk 1]
```diff
@@ -5,39 +5,54 @@
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
+    <meta
+      name="title"
+      content="Reactoads Movieapp | Explore Movies and TV Shows"
+    />
```
Comment (JorgeLVilla) [inline movie-app/public/index.html:11]: Good job on adding a title!

File: movie-app/public/manifest.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "Reactoads Movieapp",
+  "name": "Reactoads Movieapp | Explore Movies and TV Shows",
+  "description": "A Bitwise React Apprenticeship project using React.js to generate movies and TV shows using the OMDb API.",
```
Comment (JorgeLVilla) [inline movie-app/public/manifest.json:4]: Great description.

**Review by JayDampitan — APPROVED** [review]
Great usage of icon by the website title. Very clean code.

**Review by JorgeLVilla — APPROVED** [review]
Great job refactoring! Looks good to me!

**Pull request merged**